addons/sys_rack/CfgVehicles.hpp

```diff
@@ -147,6 +147,10 @@ class CfgVehicles {
         };
     };
 
+    class ParachuteBase : Helicopter {
+        class AcreIntercoms {};
```

Comment:
Shouldn't it be  `class AcreRacks{};`

Reply:
Derp, I copied and pasted for the format and forgot to change the class.